### PR TITLE
POA WUFrequency support for MonthlyByWeekAndDay

### PR DESCRIPTION
**Pull request by adibalwani**
Patch Orchestration support for specifying day-of-week for monthly frequency
https://github.com/microsoft/Service-Fabric-POA/issues/59

### CODE REVIEW

File: src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/ServiceSettings.cs [hunk 1]
```diff
@@ -161,6 +163,26 @@ private void ParseWUFrequency()
                     }
                     break;
 
+                case Frequency.MonthlyByWeekAndDay:
```
Comment (anantshankar17) [inline src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/ServiceSettings.cs:166]: Please make sure you test specifying with multiple values like "3, Monday, 00:00:00". 
Specify the day to tomorrow (4, Thursday, 09:00:00) and check if the update is triggered by POA tomorrow.

Comment (adibalwani) [reply src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/ServiceSettings.cs:166]: Completed testing on MonthlyByWeekAndDay,4,Friday,19:45:00' and 'MonthlyByWeekAndDay,4,Monday,00:00:00'

Sample from POA
Last patching attempt happened at : 4/27/2020 12:00:12 AM, Next patching cycle is scheduled at : 5/27/2020 12:00:00 AM

Comment (adibalwani) [reply src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/ServiceSettings.cs:166]: Next patching attempt doesn't look right. I'll fix it

Comment (adibalwani) [reply src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/ServiceSettings.cs:170]: Sample from MonthlyByWeekAndDay,4,Tuesday,19:05:00
Last patching attempt happened at : 4/28/2020 7:09:01 PM, Next patching cycle is scheduled at : 5/26/2020 7:05:00 PM

**Pull request merged**